src/bridge/cockpitrouter.c

```diff
@@ -615,19 +615,17 @@ strv_to_json_array (gchar **strv)
 
 static void
 add_dynamic_args_to_array (gchar ***key,
-                           gchar **config_args,
+                           const gchar **config_args,
                            JsonObject *options)
 {
   GPtrArray *arr = NULL;
-  gint length;
   gint i;
 
   g_assert (config_args != NULL);
   g_assert (key != NULL);
 
   arr = g_ptr_array_new ();
-  length = g_strv_length (config_args);
-  for (i = 0; i < length; i++)
+  for (i = 0; config_args[i]; i++)
```

Comment:
So this one breaks the loop when `config_args[i]` is `null`, right? Is it given that `config_args` cannot contain anything else, that would be "falsy"?

Reply:
The only value of a pointer that evaluates false is NULL.

Reply:
Worth noting the implementation of `g_strv_length()` itself:

```
guint
g_strv_length (gchar **str_array)
{
  guint i = 0;

  g_return_val_if_fail (str_array != NULL, 0);

  while (str_array[i])
    ++i;

  return i;
}
```

Reply:
makes sense, thanks!